frontend/views/containers/proposals/ProposalTemplate.vue

```diff
@@ -18,14 +18,17 @@
     )
       slot
 
-      label.field(v-if='isReasonStep' key='reason')
-        i18n.label Why are you proposing this change?
-        textarea.textarea(
-          ref='reason'
-          maxlength='500'
-          data-test='reason'
-        )
-        i18n.helper This is optional.
+      template(v-if='isReasonStep')
+        slot(name='reason')
```

Comment:
This looks like a significant change, did you double-check everything related to this to make sure it still works?

Also, I don't know if this bug is related to this, or to a different recent PR having to do with the `Enter` key being pressed that I didn't catch in my review (#1360), but when I created a group with 1 user, and went to create a proposal to change the mincome, and pressed Enter after entering the new mincome, this error came up:

<img width="1389" alt="Screen Shot 2022-07-23 at 5 07 19 PM" src="https://user-images.githubusercontent.com/138706/180596562-dd16b173-9c85-423f-b36d-2ff037dfd02f.png">

Do you think you could track it down?